README.md

```diff
@@ -1,27 +1,133 @@
 ## Audit scanner
 
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/kubewarden-controller)](https://artifacthub.io/packages/helm/kubewarden/kubewarden-controller)
+[![codecov](https://codecov.io/gh/kubewarden/audit-scanner/graph/badge.svg?token=EDPPGWJFSK)](https://codecov.io/gh/kubewarden/audit-scanner)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7439/badge)](https://www.bestpractices.dev/projects/7439)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Faudit-scanner.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Faudit-scanner?ref=badge_shield&issueType=license)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Fkubewarden-controller.svg?type=shield&issueType=security)](https://app.fossa.com/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Fkubewarden-controller?ref=badge_shield&issueType=security)
 
-> **Note well:** don't forget to checkout [Kubewarden's documentation](https://docs.kubewarden.io)
+> **Note well:** don't forget to checkout [Kubewarden's Audit Scanner documentation](https://docs.kubewarden.io/explanations/audit-scanner)
```

Comment:
I would prefer to use 'look at' rather than 'checkout'. To me, 'checkout' has code control connotations as well as being a little urban slang, like, 'Hey, check this out man!'

'See' is also good, maybe just the following is sufficient.

```suggestion
> **Note**: see [Kubewarden's Audit Scanner documentation](https://docs.kubewarden.io/explanations/audit-scanner)
```

I don't think the well is required in 'Note well', it's already in bold.

Just suggestions, thanks.